src/libraries/System.Private.CoreLib/src/System/Buffers/Text/FormattingHelpers.CountDigits.cs

```diff
@@ -24,8 +24,10 @@ public static int CountDigits(ulong value)
             ];
             Debug.Assert(log2ToPow10.Length == 64);
 
+            // We use a native-sized integer
+            // because it gives us a free early zero-extension to 64 bits when running on a 64-bit platform.
```

Comment:
can you remove this comment? I don't think it adds any value besides telling that JIT can't do something. This piece of code is already has a TODO to be replaced with a normal array access once https://github.com/dotnet/runtime/issues/79257 is closed